src/harness/harnessLanguageService.ts

```diff
@@ -107,7 +107,7 @@ namespace Harness.LanguageService {
         }
     }
 
-    class DefaultHostCancellationToken implements ts.HostCancellationToken {
+    export class DefaultHostCancellationToken implements ts.HostCancellationToken {

```

Comment:
I can't reproduce this anymore, but I was getting compile errors concerning the visibility of these classes. I might have brainfarted and forgot to `jake local` before `jake tests`. I'll pull this commit out.